data_pipelines/assets/flood/upstream.py

```diff
@@ -0,0 +1,24 @@
+from dagster import AssetExecutionContext, asset
+import os
+import httpx
+from data_pipelines.utils.flood.config import UPSTREAM_URL
+
+
+@asset(key_prefix=["flood"], compute_kind="xarray", io_manager_key="netcdf_io_manager")
```

Comment:
Small nitpick: this asset doesn't use xarray, so I think the compute_kind argument can be removed. This isn't very important as the only effect of the compute_kind argument is a badge by the asset in the UI.